client/src/components/SignForms/Signin.jsx

```diff
@@ -0,0 +1,114 @@
+import { useState } from 'react';
+import {
+  Button, TextField, Box, Snackbar,
+} from '@material-ui/core';
+import MuiAlert from '@material-ui/lab/Alert';
+import axios from 'axios';
+import './style.css';
+
+function Signin() {
+  const [isError, setIsError] = useState(false);
+  const [error, setError] = useState('');
+  const [isSuccess, setIsSuccess] = useState(false);
+  const [success, setSuccess] = useState('');
+  const [values, setValues] = useState({ password: '', email: '' });
+
+  const handleSubmit = (e) => {
+    if (values.password && values.email) {
+      e.preventDefault();
+      axios
+        .post('/signin', {
+          email: values.email,
+          password: values.password,
+        })
+        .then((response) => response.data)
+        .then((data) => {
+          if (data.Error) {
+            setIsSuccess(false);
+            setSuccess('');
+            setIsError(true);
+            setError(data.Error);
+          } else {
+            setIsError(false);
+            setError('');
+            setIsSuccess(true);
+            setSuccess(data.msg);
+          }
+        })
+        .catch(() => {
+          setIsSuccess(false);
+          setSuccess('');
+          setIsError(true);
+          setError('Error');
```

Comment:
as you sending error from your server, receive it from catch and display a descriptive error to user